pvlib/pvsystem.py

```diff
@@ -2810,6 +2858,71 @@ def pvwatts_losses(soiling=2, shading=3, snow=0, mismatch=2, wiring=2,
     return losses
 
 
+def dc_ohms_from_percent(V_mp_ref, I_mp_ref, dc_ohmic_percent,
+                         modules_per_string=1,
+                         strings_per_inverter=1):
+    """
+    Calculates the equivalent resistance of the wires from a percent
+    ohmic loss at STC.
+
+    Equivalent resistance is calculated with the function:
+
+    .. math::
+        Rw = (L_{stc} / 100) * (Varray / Iarray)
+
+    :math:`L_{stc}` is the input dc loss as a percent, e.g. 1.5% loss is
+    input as 1.5
```

Comment:
I suggest describing the other three variables here as well (maybe briefly saying `V_array` is `mods_per_string * v_mp_mod`, and the equivalent for `I_array`), and maybe moving the 1.5% example to the parameter description section below.